WebRoot/js/zimbraMail/core/ZmOperation.js

```diff
@@ -185,7 +185,7 @@ function() {
 		AjxCallback.simpleClosure(function(parent) {
 			ZmOperation.addDeferredMenu(ZmOperation.addSearchMenu, parent, true);
 		}));
-	ZmOperation.registerOp(ZmId.OP_SEND, {textKey:"send", tooltipKey:"sendTooltip", image:"Send", shortcut:ZmKeyMap.SEND});
+	ZmOperation.registerOp(ZmId.OP_SEND, {textKey:"send", tooltipKey:"sendTooltip", image:"Send", shortcut:ZmKeyMap.SEND, showImageInToolbar: true, showTextInToolbar: true});
```

Comment:
I think NO. When the preference MAIL_SEND_LATER_ENABLED as true, we show 'Send Later' as a menu option and not in the toolbar. The above changes deal with showing images/ text in toolbar.